python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -290,12 +352,12 @@ def flatBiasIngest(repo, calib_repo, calibdatafiles):
     try:
         calibIngestTask.run(parsedCmd)
     except sqlite3.IntegrityError as detail:
-        print('sqlite3.IntegrityError: ', detail)
-        print('(sqlite3 doesn\'t think all the calibration files are unique)')
+        log.error('sqlite3.IntegrityError: ', detail)
+        log.error('(sqlite3 doesn\'t think all the calibration files are unique)')
         flatBias_metadata = None
```

Comment:
Umm, `IntegrityError` sounds serious. Are you sure you can ignore it?

(Also, I think your first error message will redundantly give the name of the exception twice.)